src/hooks/useBatchTransaction.ts

```diff
@@ -42,7 +42,7 @@ const getCadenceBatchTransaction = (chainId: number) => {
   return `
 import EVM from ${evmAddress}
 
-transaction(calls: [{String: String}]) {
+transaction(calls: [{String: String}], mustPass: Bool) {
```

Comment:
Is there room for a value parameter? Some calls may need value transmitted as well